tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -140,32 +154,52 @@ void ROOT::Experimental::RNTupleReader::PrintInfo(const ENTupleInfo what, std::o
 }
 
 
+ROOT::Experimental::RNTupleReader *ROOT::Experimental::RNTupleReader::GetDisplayReader()
+{
+   if (!fDisplayReader)
+      fDisplayReader = Clone();
```

Comment:
It can fail because `Clone()` acquires system I/O resources.  I think it would be appropriate to use the new `RError`/`RResult`/`RException` facility here.  I would like to do this across the RNTuple code base in a separate PR.